manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -149,9 +149,6 @@ static function (ContainerBuilder $container) use ($loader): void {
                 } else {
                     $loader->load('@ContaoManagerBundle/Resources/skeleton/config/config_prod.yml');
                 }
-
-                $container->setParameter('container.autowiring.strict_mode', true);
```

Comment:
We must not remove this parameter I think. See https://github.com/symfony/recipes/pull/241/files (the change is in the flex recipe only).